src/index.js

```diff
@@ -0,0 +1,68 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
- Please, let's try to write codes that conform to the eslint as it helps us write readable and maintainable code. Please, remove and fix the linter issue